auth/src/ios/auth_ios.mm

```diff
@@ -608,5 +608,23 @@ void DisableTokenAutoRefresh(AuthData *auth_data) {}
 void InitializeTokenRefresher(AuthData *auth_data) {}
 void DestroyTokenRefresher(AuthData *auth_data) {}
 
+AuthError Auth::UseUserAccessGroup(const char* access_group_str) {
+  if (!auth_data_) {
+    return kAuthErrorUninitialized;
+  }
+  NSString* access_group_ns_str = nil;
+  if (access_group_str != nullptr && strlen(access_group_str) > 0) {
```

Comment:
If there is a blank string, let's pass it in verbatim instead of nil. Remember to update the docs to reflect this.